2214/2/battler.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'bad_words_counter'
+require_relative 'words_in_round_counter'
+require_relative 'battle'
+
+class Battler
+  BATTLES_FOLDER = 'Battles'.freeze
+  attr_reader :name, :battles
+  def initialize(name, battles)
+    @name = name
+    @battles = battles
+  end
+
+  def number_of_battles
+    @battles.size
+  end
+
+  def number_of_bad_words
+    BadWordsCounter.new(@battles).count
+  end
+
+  def bad_words_per_round
+    (number_of_bad_words.to_f / number_of_battles).round(2)
+  end
+
+  def average_number_of_words
+    WordsInRoundCounter.new(@battles).count
```

Comment:
Do we really need a separate class for this? Can it be just a method of the `Battle` class?